src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Delete.cs

```diff
@@ -100,6 +170,16 @@ public static partial class HttpClientJsonExtensions
             return DeleteFromJsonAsyncCore(taskResponse, jsonTypeInfo, cancellationToken);
         }
 
+        /// <summary>
+        /// Sends a DELETE request to the specified Uri and returns the value that results from deserializing the response body as JSON in an asynchronous operation.
+        /// </summary>
+        /// <typeparam name="TValue">The target type to deserialize to.</typeparam>
+        /// <param name="client">The client used to send the request.</param>
+        /// <param name="requestUri">The Uri the request is sent to.</param>
+        /// <param name="jsonTypeInfo">Source generated JsonTypeInfo to control the behavior during deserialization.</param>
```

Comment:
Can JsonTypeInfo be described in natural language? Something like:

```suggestion
        /// <param name="jsonTypeInfo">Source generated JSON type information to control the behavior during deserialization.</param>
```

I didn't suggest a `see cref` because the type of the parameter already shows up next to the parameter name.